Marlin/src/feature/runout.cpp

```diff
@@ -51,7 +51,7 @@ void FilamentSensorBase::filament_present(const uint8_t extruder) {
   uint8_t FilamentSensorEncoder::motion_detected;
 #endif
 
-#if FILAMENT_RUNOUT_DISTANCE_MM > 0
+#ifdef FILAMENT_RUNOUT_DISTANCE_MM
```

Comment:
Allowing for a negative or zero `FILAMENT_RUNOUT_DISTANCE_MM`?

Reply:
Definitely zero. The idea is a vendor like ourselves can set this to zero, to enable the code, while still allowing the user to change it using M425 or LCD.

For negative, no, that is not expected.